From: SJ Park <sj@kernel.org>
Subject: mm/damon/lru_sort: ignore damon_stop() return value
Date: Mon, 6 Jul 2026 07:06:21 -0700

damon_stop() return value is guaranteed to be 0.  Ignore it.

Link: https://lore.kernel.org/20260706140628.87414-7-sj@kernel.org
Signed-off-by: SJ Park <sj@kernel.org>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/damon/lru_sort.c |    6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

--- a/mm/damon/lru_sort.c~mm-damon-lru_sort-ignore-damon_stop-return-value
+++ a/mm/damon/lru_sort.c
@@ -410,8 +410,10 @@ static int damon_lru_sort_turn(bool on)
 {
 	int err;
 
-	if (!on)
-		return damon_stop(&ctx, 1);
+	if (!on) {
+		damon_stop(&ctx, 1);
+		return 0;
+	}
 
 	err = damon_lru_sort_apply_parameters();
 	if (err)
_
